docs/tutorial/t-manage-passwords.md

```diff
@@ -156,7 +166,7 @@ zookeeper-k8s/1*    active    idle   10.1.36.86
 zookeeper-k8s/2     active    idle   10.1.36.85
 ```
 
-> **Note** The operations above would also apply to charmed applications that implement  the `kafka_client` relation, for which password rotation and user deletion can be achieved in the same consistent way.
+> **Note** The operations above would also apply to Charmed applications that implement  the `kafka_client` relation, for which password rotation and user deletion can be achieved in the same consistent way.
```

Comment:
as elsewhere, I believe we use `charmed` as adjective, for which I would not capitalize it.